packages/rocketchat-slackbridge/slackbridge.js

```diff
@@ -37,6 +30,14 @@ class SlackBridge {
 		RocketChat.settings.get('SlackBridge_ExcludeBotnames', (key, value) => {
 			this.excludeBotnames = value;
 		});
+
+		RocketChat.settings.get('SlackBridge_Enabled', (key, value) => {
```

Comment:
Moved this to the bottom of the constructor to make sure all the setup was completed first.